hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/NameNodeHttpServer.java

```diff
@@ -159,9 +158,8 @@ void start() throws IOException {
       httpServer.setAttribute(DFSConfigKeys.DFS_DATANODE_HTTPS_PORT_KEY,
           datanodeSslPort.getPort());
     }
-    String httpKeytab = conf.get(DFSUtil.getSpnegoKeytabKey(conf,
-        DFSConfigKeys.DFS_NAMENODE_KEYTAB_FILE_KEY));
-    initWebHdfs(conf, bindAddress.getHostName(), httpKeytab, httpServer,
+
+    initWebHdfs(conf, httpServer,
         NamenodeWebHdfsMethods.class.getPackage().getName());
```

Comment:
this even doesn't look properly formatted

Reply:
Thanks for review. I will format the code later.

Reply:
@ayushtkn I've formatted the code, please take a review when you have time.Thanks.